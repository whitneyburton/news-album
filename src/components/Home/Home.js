import React from 'react';
import mountains from '../../images/mountains.jpg';

export const Home = () => {
  return (
    <div className='Home'>
      <h1>Home</h1>
      {mountains}
    </div>
  )
}