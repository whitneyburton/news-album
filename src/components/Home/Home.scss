.Home {
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  background-image: url('../images/mountains.jpg');
  height: 65vh;
  background-size: cover;

  h1 {
    text-align: left;
    color: #fff;
    width: 75%;
    text-shadow: 2px 2px 10px #000;
  }
}