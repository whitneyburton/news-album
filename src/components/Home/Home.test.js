import React from 'react';
import { shallow } from 'enzyme';
import { Home } from './Home';

describe('Home', () => {
  it('should match the snapshot', () => {
    const wrapper = shallow(<Home />);
    expect(wrapper).toMatchSnapshot();
  });
});