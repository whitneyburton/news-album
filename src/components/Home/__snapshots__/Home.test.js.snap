// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should match the snapshot 1`] = `
<div
  className="Home"
>
  <h1>
    welcome! newsalbum gathers the ten latest STEM articles from top sources for you to have at your fingertips. click on a news source above to get started.
  </h1>
</div>
`;
